terraform/sitesusa.app.cloud.gov.tf

```diff
@@ -0,0 +1,384 @@
+/*
+    To keep this repo clean and organized, we are keeping all of the sitesusa
+    dns records here.
+ */
+ resource "aws_route53_zone" "sitesusa_app_cloud_gov_zone" {
```

Comment:
This variable name `sitesusa_app_cloud_gov_zone` needs to be referenced in the `zone_id` sections below.